src/workflow/mod.rs

```diff
@@ -314,12 +316,18 @@ pub mod managed_wf {
         pub(crate) fn complete_local_activity(
             &mut self,
             seq_num: u32,
-            result: ActivityResult,
+            result: ActivityExecutionResult,
         ) -> Result<()> {
             self.mgr.notify_of_local_result(
                 seq_num,
                 LocalResolution::LocalActivity {
-                    result,
+                    // We accept normal execution results and do this conversion because there
+                    // are more helpers for constructing them.
+                    result: result
```

Comment:
Should this not return a meaningful error back instead of panicking?

Reply:
This is test code

Reply:
`ManagedWFFunc` is test only